examples/esp_netif/multiple_netifs/sdkconfig.defaults

```diff
@@ -1,2 +1,5 @@
+# You can use CONFIG_ESP_NETIF_SET_DNS_PER_DEFAULT_NETIF
```

Comment:
Do we expect this to be the default for users? If so, we should enable it here and point in README how to disable.